keyboards/atreus/keymaps/nojjan/config.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "../../config.h"
```

Comment:
Could you remove this line? 

It's not needed, as the makefile script actually handles this automatically.